src/main/java/wenjieBot/exceptions/InvalidInputException.java

```diff
@@ -0,0 +1,14 @@
+package wenjieBot.exceptions;
+
+public class InvalidInputException extends DukeException {
```

Comment:
Seems like this exception is only for date inputs, perhaps it can be renamed to InvalidDateInputException instead?